src/Synapse/Validator/Constraints/I18n/BelongsToEntity.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Synapse\Validator\Constraints\I18n;
+
+use Synapse\Validator\Constraints\BelongsToEntity as ParentConstraint;
+
+class BelongsToEntity extends ParentConstraint
+{
+    public $message = 'DOES_NOT_BELONG_TO_ENTITY';
```

Comment:
true enough on having all the classes in one place.  as for switching to constants, if we wanted to do it right it'd be better to extend the base validator to use a getMessage method which would return the constant instead of accessing a public property.  Just having a public property allows the message to be changed and is kinda sloppy IMO.  Just keeping them as props is probably fine since we're just building on top of symfony's stuff.